enforcer-rules/src/test/java/org/apache/maven/plugins/enforcer/TestRequireJavaVersion.java

```diff
@@ -89,30 +88,36 @@ void settingsTheJavaVersionAsNormalizedVersionShouldNotFail() throws EnforcerRul
 
     @Test
     void excludingTheCurrentJavaVersionViaRangeThisShouldFailWithException() {
-        assertThrows(EnforcerRuleException.class, () -> {
-            String thisVersion = RequireJavaVersion.normalizeJDKVersion(SystemUtils.JAVA_VERSION);
-
-            RequireJavaVersion rule = new RequireJavaVersion();
-            // exclude this version
-            rule.setVersion("(" + thisVersion);
-
-            EnforcerRuleHelper helper = EnforcerTestUtils.getHelper();
-            rule.execute(helper);
-            // intentionally no assertThat(...) because we expect and exception.
-        });
-        // intentionally no assertThat(...) because we expect and exception.
+        String thisVersion = RequireJavaVersion.normalizeJDKVersion(SystemUtils.JAVA_VERSION);
+        String requiredVersion = "(" + thisVersion;
+
+        assertThatThrownBy(() -> {
+                    RequireJavaVersion rule = new RequireJavaVersion();
+                    rule.setVersion(requiredVersion);
+
+                    EnforcerRuleHelper helper = EnforcerTestUtils.getHelper();
+                    rule.execute(helper);
+                })
+                .isInstanceOf(EnforcerRuleException.class)
+                .hasMessage("The requested JDK version %s is invalid.", requiredVersion);
     }
 
     @Test
-    @Disabled
-    // TODO: Think about the intention of this test? What should it prove?
-    void thisShouldNotCrash() throws EnforcerRuleException {
-        RequireJavaVersion rule = new RequireJavaVersion();
-        rule.setVersion(SystemUtils.JAVA_VERSION);
-
-        EnforcerRuleHelper helper = EnforcerTestUtils.getHelper();
-        rule.execute(helper);
-        // intentionally no assertThat(...) because we don't expect and exception.
```

Comment:
This test has been disabled for years now and as far as I can tell it is not really testing anything that is not being already tested elsewhere, so I figured we should just delete it.